server/routes/api.js

```diff
@@ -74,15 +74,47 @@ module.exports = (services) => {
       // store trip
       const tripResult = await services.db.trip.create(trip);
       const tripid = tripResult.id;
+      console.log('Trip stored at id:', tripid);
+
+      let sourceId = 0, des_id = 0;
+      for (let i = 0, len = routes.length; i < len; i += 1) {
+        if (routes[i].name === 'source' || routes[i].name === 'destination') {
+          const route = {
+            type: routes[i].name,
+            latitude: routes[i].lat,
+            longitude: routes[i].lng,
+          };
+          // store places
+          if(routes[i].name === 'source') {
+            sourceId = (await services.db.place.create(route)).id;
+          } else {
+            desId = (await services.db.place.create(route)).id;
```

Comment:
@bill-feng I think you define as `des_id` at line 79 and `desId` here. It doesn't cause a problem as desId is used after but if you can fix it, it'll be better :)

Reply:
Fixed!